web/public/locales/en/common.json

```diff
@@ -233,7 +233,7 @@
     "unpinSuccess": "Sucess unpin",
     "tagsExplain": "Optionallly, write and press enter for each keyword that you want to add",
     "imagesLabel": "Upload the images you want",
-    "imagesExplain": "Images will be displayed in a carrousel in your _helpbutton_ page",
+    "imagesExplain": "Maximum size 4Mb. Images will be displayed in a carrousel in your _helpbutton_ page",
```

Comment:
where is this number coming from? it fully depends on the server configurations. can't be hardcoded in translations.

Reply:
This is something i can remove. it was because there was no MAX IMAGE SIZE error